src/RWAPIIndexer/Elasticsearch.php

```diff
@@ -188,10 +188,12 @@ public function getIndexAlias($index) {
    *   Elasticsearch index name.
    * @param array $mapping
    *   Index mapping.
+   * @param int $shards
+   *   The number of shards for this index.
    */
-  public function create($index, array $mapping) {
+  public function create($index, array $mapping, int $shards = 1) {
```

Comment:
For consistency with the `replicas` setting, I think the `shards` should be passed to the constructor or the opposite and have the `replicas` also passed to `createIndex`.